packages/fetch-cached-dns/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@vercel/fetch-cached-dns",
+  "version": "2.0.2",
+  "description": "A decorator on top of `fetch` that caches the DNS query of the `hostname` of the passed URL",
+  "license": "MIT",
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "util.js"
+  ],
+  "scripts": {
+    "test": "jest test",
+    "lint": "eslint .",
+    "lint:staged": "lint-staged"
+  },
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vercel/fetch-cached-dns.git"
+  },
+  "author": "Nathan Rajlich <nate@vercel.com>",
+  "pre-commit": "lint:staged",
+  "lint-staged": {
+    "*.js": [
+      "eslint",
+      "prettier --write --single-quote --no-semi",
+      "git add"
+    ]
+  },
+  "eslintConfig": {
+    "extends": "@zeit/eslint-config-node"
+  },
+  "bugs": {
+    "url": "https://github.com/vercel/fetch-cached-dns/issues"
+  },
+  "homepage": "https://github.com/vercel/fetch-cached-dns#readme",
+  "peerDependencies": {
+    "node-fetch": "*"
+  },
+  "dependencies": {
+    "@types/node-fetch": "2.3.2",
+    "@zeit/dns-cached-resolve": "2.1.2"
+  },
+  "devDependencies": {
+    "@zeit/eslint-config-node": "0.1.0",
```

Comment:
Since not all of the repos were enforcing a style guide I've removed eslint from all the packages. My next step is to rewrite everything in TS anyways and then I'll add a linter & formatter back (and maybe just rely on whatever turbo will let us enforce for all of the packages holistically).